librenms.cron

```diff
@@ -1,4 +1,5 @@
 # It's recommended not to run this cron anymore - please see librenms.nonroot.cron
+# PLEASE DO NOT USE THIS FILE
```

Comment:
The original docs, before your time :p Told people to symlink that file to /etc/cron.d/ :(